src/screens/NewTerm.js

```diff
@@ -32,6 +33,16 @@ export default ({
   const [categoryId, setCategoryId] = useState('');
   const [englishPhrase, setEnglishPhrase] = useState('');
   const [malagasyPhrase, setMalagsyPhrase] = useState('');
+  const [selectIcon, setSelectIcon] = useState(true);
```

Comment:
Since it is a boolen, I think 'selectIcon in line 36 could be renamed as 'showSelectIcon' and 'setShowSelectIcon' to make it clearer to understand